adapter/src/main/java/org/jvalue/ods/adapterservice/datasource/api/rest/v1/DatasourceEndpoint.java

```diff
@@ -20,52 +22,48 @@
 public class DatasourceEndpoint {
   private final DatasourceManager datasourceManager;
 
+  private <T> T handleErrors(CheckedSupplier<T> function) throws ResponseStatusException {
```

Comment:
duplicate with other endpoints => should we use a convert method and place it next to the mapping?

Reply:
There are two different Mappings classes though, one for the Adapter, and one for the Datasources.

I have not found a good way to avoid this duplication yet. Someone got any ideas?